rviz_rendering/src/rviz_rendering/render_system.cpp

```diff
@@ -408,10 +408,7 @@ RenderSystem::makeRenderWindow(
 
   params["currentGLContext"] = Ogre::String("false");
 
-  // This line is needed to receive mouse events on Windows, but results in a crash on OS X
-#ifdef _WIN32
   params["externalWindowHandle"] = Ogre::StringConverter::toString(window_id);
-#endif
```

Comment:
Btw, this changes the behavior on Linux as well as macOS.

Reply:
If I revert this particular change my rviz starts working again... I don't know if this is specific to my machine or not.

Reply:
> Btw, this changes the behavior on Linux as well as macOS.
In our experience the `externalWindowHandle` param is ignored on linux and does not influence the behavior.